src/wfs/src/main/java/org/geoserver/wfs/json/ComplexGeoJsonWriter.java

```diff
@@ -45,6 +46,7 @@
 
     private static Class NON_FEATURE_TYPE_PROXY;
     private static final String DATATYPE = "@dataType";
+    private static final String INSIDE_ARRAY_ATTRIBUTE = "${inside-array}";
```

Comment:
What's this?

Reply:
Adding this as comment:
A string constant for representing a not needed key name because object is
being added inside an already named json array